src/main/java/me/axieum/mcmod/authme/util/SessionUtil.java

```diff
@@ -158,6 +160,9 @@ private static void setSession(Session session) throws IllegalAccessException
     {
         // NB: Minecraft#session is a final property - use mixin accessor
         ((SetSessionMixin) MinecraftClient.getInstance()).setSession(session);
+        // Necessary for Realms to re-check for a valid session.
```

Comment:
Being really picky here, but could we add a space above and remove the period/full-stop 😛

Reply:
Nothing wrong with being picky about code style -- done! :)